src/components/button/template.njk

```diff
@@ -10,9 +21,14 @@
   {% endif %}
 {% endif %}
 
+{% if params.isStartButton %}
+  {% set iconMarkUp = '<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" height="19px" width="30px"><path fill="currentColor" d="M7.5 0H0l10 10L0 20h7.5l10-10-10-10z"/></svg>' %}
```

Comment:
I think we can improve the inline SVG by doing the following:

- remove 'data-name' attribute
- add `.govuk-button__icon` class instead of styling the 'svg' element
- change `height` and `width` to be relative units (`em`)

Reply:
First two points have been done.